### PR TITLE
Fig2.2 gauss and  quantile plot

### PR DESCRIPTION
**Pull request by karm-patel**
Hi Dr. @murphyk, Prof. @nipunbatra, and @patel-zeel, can you please review the following changes? 
## Description

1. Added `notebooks/gauss_plot.ipynb` for `scripts/gauss_plot.py`
2. Added `notebooks/quantile_plot.ipynb` for `scripts/quantile_plot.py`
3. Updated `_toc.yml` file

### Figure Number
book1 - 2.2

### Figures
[Gist](https://gist.github.com/karm-patel/18a8a37430939b831c1814d9bc8824ed)

### Issue 
#702 

## Steps

- [x] Changed the name of the variables/functions to PEP 8 style
- [x] Latexified the figures
- [x] Converted code to JAX

## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

### CODE REVIEW

File: notebooks/gauss_plot.ipynb [hunk 1]
```diff
@@ -0,0 +1,149 @@
+{
```
Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    random_var = norm
</pre><p>Let's not do this. We can use <code>norm</code> directly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.        plt.xlabel("x")
</pre><p><code>plt.xlabel("$x$")</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by murphyk** [comment]
There are 17 files in this PR... are they all needed? @patel-zeel  PTAL and merge if you approve.

**Comment by patel-zeel** [comment]
Sure, Dr. @murphyk. These changes are showing up due to a drastic change in the `master` due to the removal of Jupyter book. Effectively only two notebooks are being added to the `master`.

**Pull request merged**